docs/src/pages/components/data-grid/export/export.md

```diff
@@ -6,23 +6,40 @@ title: Data Grid - Export
 
 <p class="description">Easily export the rows in various file formats such as CSV, Excel, or PDF.</p>
 
-## CSV export
+## Enabling export
+
+### Default Toolbar
 
-The easiest way to enable the CSV export is to pass the `GridToolbar` component in the `Toolbar` [component slot](/components/data-grid/components/#toolbar).
+To enable the export selector, pass the `GridToolbar` component in the `Toolbar` [component slot](/components/data-grid/components/#toolbar).
 
 {{"demo": "pages/components/data-grid/export/ExportDefaultToolbar.js", "bg": "inline"}}
 
-### Custom toolbar
+### Custom Toolbar
+
+The export selector is provided in a stand alone component named `GridToolbarExport`. You can use it in a custom toolbar component as follow.
+
+```jsx
+function CustomToolbar() {
+  return (
+    <GridToolbarContainer>
+      <GridToolbarExport />
+    </GridToolbarContainer>
+  );
+}
+```
 
-To enable the CSV export in a custom toolbar, use the `GridToolbarExport` component.
+{{"demo": "pages/components/data-grid/export/ExportCustomToolbar.js", "bg": "inline", "defaultCodeOpen": false}}
 
-{{"demo": "pages/components/data-grid/export/ExportCustomToolbar.js", "bg": "inline"}}
+## Customize export selector
```

Comment:
I think the page has a structure issue.
Basically we have

- How to customize the UI 
  - How to give export params to the Grid
  
- How to customize the columns
- How to customize the rows 


But the "How to give export params to the Grid" is also interesting to customize the rows and the columns.

Reply:
The subsection should probably be a section (each at the same level) 

- Export options
- Disabled export
- Exported columns
...